### ISSUE TITLE
about training on icdar2015 dataset

### ISSUE THREAD

**Issue by happog** [saicoco/tf-sotd#4]
Hi,  did you train sotd on icdar dataset? I have trained it on icdar2015 but the predicted map is all background class. Should I use another loss function such as cross entropy? Thank you!

**Comment by saicoco** [comment]
Sorry to I reply to so late, I didn't train it on icdar, the backgroud, border and text area are imbalance, so you need to add OHEM or other balanced loss function to train it, then it will work well
